boilerplate.js

```diff
@@ -21,19 +21,21 @@ const finish = async function (context) {
   const { parameters, system, print, ignite } = context
   const name = parameters.third
 
-  // initial git
-  if (system.which('git')) {
-    const spinner = print.spin('configuring git')
-    ignite.log('git init .')
-    await system.run('git init .')
-    ignite.log('git add .')
-    await system.run('git add .')
-    ignite.log('git commit')
-    await system.run('git commit -m "Initial commit."')
-    // setup husky git hooks
-    spinner.text = 'setting up git hooks'
-    system.run(`node node_modules/husky/bin/install .`)
-    spinner.succeed('configured git')
+  if (parameters.options['skip-git'] === false) {
```

Comment:
I'm sorry.  I was wrong.  It's `!== true`.   

When you set --skip-git, it shows up in options as `{ 'skip-git': true }`.  When you omit it, options is `{}` and if you type --no-skip-git it becomes `{'skip-git': false}`.

I'm sorry for that.

Give it a test with & without and make sure it's doing the right thing.

Oh boy, it's late.  I'm not thinking clearly.  :)